CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Fixes
+- Fix theme switcher not applying the new theme when the component's page is changed @mnajdova ([#911](https://github.com/stardust-ui/react/pull/911))
```

Comment:
Note, the CHANGELOG should only be used for public changes to the `@stardust-ui/react` package.

Changes to the docs should not get CHANGELOG entries.  Someday, we may want to maintain a docs/CHANGELOG.